index.html

```diff
@@ -46,4 +46,238 @@ <h1 class="mola">  I'm Sayed Ramin.<br />Glad to see<br />
         </li>
     </ul>
 </div>
-</section>
\ No newline at end of file
+</section>
+<!--Works Section page 1-->
+<section class="page1">
+    <div class="@continer">
+  
+      <div class="layout1">
+        <a href="#">
+          <img
+            class="imagework"
+            src="image/works/Snapshoot Portfolio.png"
+            alt="background"
+          />
+        </a>
+        <div class="pra">
+          <h3 class="title1">Tonic</h3>
+          <ul class="flex">
+            <li class="canopy-">CANOPY</li>
+            <li class="p"> <img src="image/works/Counter.png"></li>
+            <li class="role">Back End Dev</li>
+            <li class="p"> <img src="image/works/Counter.png"></li>
+            <li class="year">2015</li>
+          </ul>
+          <p class="text1">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="btn1">
+            <li class="btn">
+              <button class="html" type="button">HTML</button>
```

Comment:
Kindly update your works section to match the figma design better🙏

Your design:
<img width="238" alt="image" src="https://github.com/sadat-Ramin/My_first_portfolio/assets/37116322/010bfca0-1230-46fb-9e45-924d2c44e5a2">

Figma design:
<img width="211" alt="image" src="https://github.com/sadat-Ramin/My_first_portfolio/assets/37116322/2858e548-b7ab-4d3e-8dd5-dfaa9441df7e">

A few things to note:
- Please update the background color of the works section. The correct color is `#f7f7f9`
- Please add the required padding between the end of the headline section and the beginning of the first card
- Kindly increase the spacing between the text elements in each card
- Kindly make sure the left and right margins are equal